bootstrap/debug.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```

Comment:
What is this script for?

Reply:
It will 
1. deploy bootstrapper.debug.yaml.
2. wait for pod kubeflow-bootstrapper-0 to be in phase 'Running'
3. run kubectl port-forward, opening port 2345 to the pod's container
4. wait - cleanup on Ctrl-C

bootstrapper.debug.yaml will start the following process in kubeflow-bootstrapper-0
"/opt/kubeflow/dlv.sh". This script runs
```
dlv --listen=:2345 --headless=true --api-version=2 exec /opt/kubeflow/bootstrapper -- --in-cluster --namespace=kubeflow --config=/opt/kubeflow/default.yaml --app-dir=/opt/bootstrap/default --registries-config-file=/opt/kubeflow/image_registries.yaml
```

[dlv](https://github.com/derekparker/delve) is a golang debugger that works with Intellij's IDE [Goland](https://www.jetbrains.com/go/)

Reply:
Can you add a comment explaining that.